src/H5Iint.c

```diff
@@ -77,6 +77,9 @@ typedef struct {
 static void * H5I__unwrap(void *object, H5I_type_t type);
 static htri_t H5I__clear_type_cb(void *_id, void *key, void *udata);
 static void * H5I__remove_common(H5I_type_info_t *type_info, hid_t id);
+static int    H5I__dec_ref(hid_t id, void **token);
+static int    H5I__dec_app_ref(hid_t id, void **token);
+static int    H5I__dec_app_ref_always_close(hid_t id, void **token);
```

Comment:
why are these named with a token? that's a request right? or I'm missing something?

Reply:
Ah, yes, request would be better.  Fixed.